packages/@aws-cdk/pipelines/lib/codepipeline/private/logging.ts

```diff
@@ -0,0 +1,18 @@
+import * as codebuild from '@aws-cdk/aws-codebuild';
+
+export function mergeLoggings(a: codebuild.LoggingOptions, b?: codebuild.LoggingOptions): codebuild.LoggingOptions | undefined;
+export function mergeLoggings(a: codebuild.LoggingOptions | undefined, b: codebuild.LoggingOptions): codebuild.LoggingOptions | undefined;
+export function mergeLoggings(a?: codebuild.LoggingOptions, b?: codebuild.LoggingOptions): codebuild.LoggingOptions | undefined;
+export function mergeLoggings(a?: codebuild.LoggingOptions, b?: codebuild.LoggingOptions) {
```

Comment:
Add spacing between these for my delicate eyeballs, please.

Reply:
Granted, with my suggestions above, these might not be needed.

Reply:
Is there any suggestion for a coding style?

Reply:
you know, I didn't even notice that it was an overloaded function because I reacted too quickly to the spacing issue. This is appropriate and my eyeballs will deal, lol.